changelog.mdx

```diff
@@ -7,6 +7,37 @@ description: Updates to Chalk!
 
 ---
 
+## November 4, 2024
+
+### Offline Query Specifications for shards and workers
+
+When running an asynchronous offline query, you can now specify the inputs `num_shards` and `num_workers` as
+parameters to allow for more granular control over the parallelization of your query execution. To see all of your
+offline query options, check out the [offline query documentation](/api-docs#ChalkClient.offline_query).
+
+In addition, offline query progress reporting now specifies progress by shard, to give developers more insight
+into where their offline query is in the execution progress.
+
+### ChalkClient can now use the default Github branch
+
+You can now default to using the name of your current Github branch when developing using the `ChalkClient`.
```

Comment:
```suggestion
You can now default to using the name of your current Github branch when developing using the `ChalkClient`.
```
Nit but does this also work with Gitlab? Is it just the git branch?

Reply:
ah should be both...